synapse/server_notices/server_notices_manager.py

```diff
@@ -100,7 +111,7 @@ async def get_or_create_notice_room_for_user(self, user_id: str) -> str:
             user_id: complete user id for the user we want a room for
 
         Returns:
-            room id of notice room.
+            room id of notice room and flag indicating weather room was created.
```

Comment:
```suggestion
            room id of notice room and flag indicating whether room was created.
```

Reply:
Text removed since the method does no longer return a flag after following [this suggestion](https://github.com/matrix-org/synapse/pull/12115#discussion_r823054570)